pom.xml

```diff
@@ -65,6 +65,14 @@
                     <target>17</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>16</source>
+                    <target>16</target>
+                </configuration>
+            </plugin>
```

Comment:
It's already present above, why do you need to duplicate it?

Reply:
Sorry Sir, I am new to github, doing PR's, can you please help me out with how can i update only the code which is latest change ? And not the entire thing like this happened...

Reply:
See `git add` command if that's what you're talking about